pkgs/build-support/trivial-builders/test/invoke-writeDirectReferencesToFile.nix

```diff
@@ -0,0 +1,4 @@
+{ pkgs ? import ../../../.. { config = {}; overlays = []; } }:
```

Comment:
I know this is not the favorite way to inject packages (which would be `callPackage` etc), but this is nicer because it allows quick manual testing as well.
Cross compilation, where `callPackage` is relevant, is not a concern here.